src/sqlancer/postgres/oracle/PostgresCERTOracle.java

```diff
@@ -0,0 +1,216 @@
+        }
+        return increase;
+    }
+
+    @Override
+    protected boolean mutateGroupBy() {
+        boolean increase = select.getGroupByExpressions().size() > 0;
+        if (increase) {
+            select.clearGroupByExpressions();
+        } else {
+            select.setGroupByExpressions(select.getFetchColumns());
+        }
+        return increase;
+    }
+
+    @Override
+    protected boolean mutateHaving() {
+        if (select.getGroupByExpressions().size() == 0) {
+            select.setGroupByExpressions(select.getFetchColumns());
+            select.setHavingClause(gen.generateExpression(0, PostgresDataType.BOOLEAN));
+            return false;
+        } else {
+            if (select.getHavingClause() == null) {
+                select.setHavingClause(gen.generateExpression(0, PostgresDataType.BOOLEAN));
+                return false;
+            } else {
+                select.setHavingClause(null);
+                return true;
+            }
+        }
+    }
+
+    @Override
+    protected boolean mutateAnd() {
+        if (select.getWhereClause() == null) {
+            select.setWhereClause(gen.generateExpression(0, PostgresDataType.BOOLEAN));
+        } else {
+            PostgresExpression newWhere = new PostgresBinaryLogicalOperation(select.getWhereClause(),
+                    gen.generateExpression(0, PostgresDataType.BOOLEAN), BinaryLogicalOperator.AND);
+            select.setWhereClause(newWhere);
+        }
+        return false;
+    }
+
+    @Override
+    protected boolean mutateOr() {
+        if (select.getWhereClause() == null) {
+            select.setWhereClause(gen.generateExpression(0, PostgresDataType.BOOLEAN));
+            return false;
+        } else {
+            PostgresExpression newWhere = new PostgresBinaryLogicalOperation(select.getWhereClause(),
+                    gen.generateExpression(0, PostgresDataType.BOOLEAN), BinaryLogicalOperator.OR);
+            select.setWhereClause(newWhere);
+            return true;
+        }
+    }
+
+    private int getRow(SQLGlobalState<?, ?> globalState, String selectStr, List<String> queryPlanSequences)
+            throws AssertionError, SQLException {
+        int row = -1;
+        String explainQuery = "EXPLAIN " + selectStr;
+
+        if (globalState.getOptions().logEachSelect()) {
+            globalState.getLogger().writeCurrent(explainQuery);
+            try {
+                globalState.getLogger().getCurrentFileWriter().flush();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+
+        // Get row count
+        SQLQueryAdapter q = new SQLQueryAdapter(explainQuery, errors);
+        try (SQLancerResultSet rs = q.executeAndGet(globalState)) {
+            if (rs != null) {
+                while (rs.next()) {
+                    String content = rs.getString(1).trim();
+                    if (content.contains("rows")) {
```

Comment:
"row="

Reply:
```                                     QUERY PLAN
-------------------------------------------------------------------​------------------
 Limit  (cost=0.29..14.48 rows=2 width=244)
   ->  Index Scan using tenk1_unique2 on tenk1  (cost=0.29..71.27 rows=10 width=244)
         Index Cond: (unique2 > 9000)
         Filter: (unique1 < 100)
```
According to these query plans I think going with "rows=" || "row=" would be better.

Reply:
Sorry, my typo. You can just use "rows=" I think